app/views/forms/_odk_group.xml.erb

```diff
@@ -0,0 +1,20 @@
+<group appearance="field-list">
+  <label><%= qing.first.parent.group_name %></label>
+  <% grid_mode = grid_mode?(qing) %>
+
+  <% if grid_mode %>
+    <%= render('odk_questioning',
+      qing: qing.first,
+      style: style,
+      grid_mode: grid_mode,
+      options: {appearance: 'label'}) %>
```

Comment:
This looks like an options hash inside an options hash?

Perhaps just `label_row: true` here, and then construct the options hash for the odk_input_tag method inside the partial? Else it's kind of confusing.